orchagent/pfc_detect_mellanox.lua

```diff
@@ -18,9 +18,11 @@ local timestamp_struct = redis.call('TIME')
 local timestamp_current = timestamp_struct[1] + timestamp_struct[2] / 1000000
 local timestamp_string = tostring(timestamp_current)
 redis.call('HSET', 'TIMESTAMP', 'pfcwd_poll_timestamp_last', timestamp_string)
-local real_poll_time = poll_time
+local effective_poll_time = poll_time
+local effective_poll_time_lasttime = redis.call('HGET', 'TIMESTAMP', 'effective_pfcwd_poll_time_last')
```

Comment:
What's the return value if `effective_pfcwd_poll_time_last` is not present in db?

Reply:
A boolean `false` in case it doesn't exist in the db.
It should exist after the first call of the plugin.